timemachine/cpp/src/local_md_utils.cu

```diff
@@ -62,17 +62,18 @@ construct_ixn_group_potential(const int N, std::shared_ptr<Potential> pot, const
     gpuErrchk(cudaMemcpy(&h_params[0], d_params, P * sizeof(*d_params), cudaMemcpyDeviceToHost));
     std::vector<int> dummy_idxs{0};
     std::vector<int> shape{P};
+    std::optional<std::set<int>> dummy_col_idxs(std::nullopt);
 
     if (std::shared_ptr<NonbondedAllPairs<float>> nb_pot = std::dynamic_pointer_cast<NonbondedAllPairs<float>>(pot);
         nb_pot) {
         std::shared_ptr<Potential> ixn_group(
-            new NonbondedInteractionGroup<float>(N, dummy_idxs, nb_pot->get_beta(), nb_pot->get_cutoff()));
+            new NonbondedInteractionGroup<float>(N, dummy_idxs, dummy_idxs, nb_pot->get_beta(), nb_pot->get_cutoff()));
```

Comment:
It looks like this should fail due to this check
https://github.com/proteneer/timemachine/blob/split_ff_solv_ixn_gpu/timemachine/cpp/src/nonbonded_interaction_group.cu#L63-L68